Python/Product/PythonTools/PythonTools/OutliningTaggerProvider.cs

```diff
@@ -323,7 +322,7 @@ class OutliningWalker : PythonWalker {
             }
 
             public override bool Walk(ClassDefinition node) {
-                AddTagIfNecessary(node, node.HeaderIndex, node.Decorators);
+                AddTagIfNecessary(node, node.HeaderIndex + 1, node.Decorators);

```

Comment:
I'll merge this fix, but go ahead and make that issue. It's probably too risky to mess with the AST at this stage, but after 2.2 it'll be fine.